crates/turbopack-css/src/process.rs

```diff
@@ -731,14 +732,29 @@ async fn process_content(
 /// ```
 ///
 /// is wrong for a css module because it doesn't have a class name.
-struct CssValidator {
+#[derive(Default)]
+struct CssModuleValidator {
     errors: Vec<CssError>,
+    used_grid_areas: HashSet<String>,
```

Comment:
```suggestion
    used_grid_areas: IndexSet<String>,
```

Errors should be deterministic